languages/python/templates/setup.mustache

```diff
@@ -10,21 +10,31 @@ NAME = "{{{projectName}}}"
 VERSION = "{{packageVersion}}"
 {{#apiInfo}}
 {{#apis}}
-{{^hasMore}}
+{{#-last}}
 # To install the library, run the following
 #
 # python setup.py install
 #
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["urllib3 >= 1.15", "six >= 1.10", "certifi", "python-dateutil", "fds.protobuf.stach < 2.0.0", "pandas < 2.0.0"]
+REQUIRES = [
+  "urllib3 >= 1.25.3",
+  "python-dateutil",
+  "fds.protobuf.stach.extensions < 2.0.0",
+  "pandas < 2.0.0",
+  "openpyxl >= 3.0.0",
```

Comment:
@FullstackGJJ 

Can we remove pandas since it is included as part of stach extension
Is openpyxl only needed for examples? If yes, can we remove it from SDK generator and only add it to the requirements.txt file for examples if its not already included

Reply:
Done